vc-verifier/kotlin/vcverifier/src/main/java/io/mosip/vercred/vcverifier/utils/CBORConverter.kt

```diff
@@ -0,0 +1,68 @@
+package io.mosip.vercred.vcverifier.utils
+
+import co.nstant.`in`.cbor.CborDecoder
+import co.nstant.`in`.cbor.model.*
+import co.nstant.`in`.cbor.model.Map
+import java.io.ByteArrayInputStream
+import kotlin.Any
+import kotlin.Array
+import kotlin.Boolean
+import kotlin.ByteArray
+import kotlin.IllegalArgumentException
+import kotlin.Int
+import kotlin.Long
+import kotlin.String
+
+
+class CBORConverter {
```

Comment:
This should be reused from pixelpass library

Reply:
Its unusued, it has been removed now